src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,100 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private long id;
+
+  @Column(nullable = false)
```

Comment:
You can either do this like this, or only using @NotNull annotation instead of the @Column annotation that here is only used for passing that specific behaviour as an annotation attribute.
I usually use @NotNull